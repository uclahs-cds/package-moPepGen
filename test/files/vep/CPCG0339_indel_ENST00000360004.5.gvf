##fileformat=VCFv4.2
##mopepgen_version=0.0.1
##parser=parseVEP
##reference_index=/data/Index/GRCh38-EBI-GENCODE34
##genome_fasta=
##annotation_gtf=
##source=gindel
##CHROM=<Description='Gene ID'>
##INFO=<ID=TRANSCRIPT_ID,Number=1,Type=String,Description="Transcript ID">
##INFO=<ID=GENE_SYMBOL,Number=1,Type=String,Description="Gene Symbol">
##INFO=<ID=GENOMIC_POSITION,Number=1,Type=String,Description="Genomic Position">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
ENSG00000196126.11	5493	INDEL-5493-G-GTAT	G	GTAT	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32584355-32584356;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	5494	INDEL-5494-AGGG-A	AGGG	A	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32584352-32584354;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	5526	INDEL-5526-GGTGCGG-G	GGTGCGG	G	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32584317-32584322;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	5664	INDEL-5664-G-GGA	G	GGA	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32584184-32584185;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	5665	INDEL-5665-C-CGG	C	CGG	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32584183-32584184;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	5670	INDEL-5670-GC-G	GC	G	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32584178;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	5670	INDEL-5670-GCG-G	GCG	G	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32584177-32584178;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	5677	INDEL-5677-GC-G	GC	G	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32584171;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	8223	INDEL-8223-C-CA	C	CA	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32581625-32581626;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	8226	INDEL-8226-AG-A	AG	A	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32581622;GENE_SYMBOL=HLA-DRB1
ENSG00000196126.11	8280	INDEL-8280-A-ATG	A	ATG	.	.	TRANSCRIPT_ID=ENST00000360004.5;GENOMIC_POSITION=chr6:32581568-32581569;GENE_SYMBOL=HLA-DRB1
