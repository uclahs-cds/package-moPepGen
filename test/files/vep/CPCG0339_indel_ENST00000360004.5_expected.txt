AGWTPTADTTTGLWR
ASQCSGESNLR
AVTELGRPDAEYWNSQKDILEDR
AVTELGRPDAEYWNSQKDILEDRR
AVTELGRPDAEYWNSQKDILEQAGR
AVTELGRPDAEYWNSQKDILER
AVTELGRPDAEYWNSQKDILERR
CHFFNGTER
CHFFNGTERFLDR
CHFFNGTERFLDRYFYNQEESVR
CHFFNGTERVR
CHFFNGTERVRFLDR
DILEDRR
DILEDRRAAVDTYCR
DILEDRRAGWTPTADTTTGLWR
DILEDRRGPGGHLLQTQLR
DILEDRRGPR
DILEDRRGR
DILEQAGPGGHLLQTQLR
DILEQAGPR
DILEQAGPRWTPTADTTTGLWR
DILEQAGR
DILEQAGRGGHLLQTQLR
DILEQAGRVDTYCR
DILEQARAGWTPTADTTTGLWR
DILERRR
DKPSHAVEWR
DKPSHAVEWRAR
DKPSHAVEWRARSESAQSK
DKPSHSGMESTV
ECHFFNGTERFLDR
ECHFFNGTERFLDRYFYNQEESVR
FLWQPKK
FLWQPKKCHFFNGTER
FLWQPKYR
FLWQPKYRECHFFNGTER
FTPAKWSTQA
GCGELHSAAASPT
GGHLLQTQLR
GGHLLQTQLRGCGELHSAAASPT
GLHLPSGAPK
GLHLPSGAPKR
GLHLPSGAPKRDKPSHAVEWR
GLHLPSGAPKRDKPSHSGMESTV
GPGGHLLQTQLR
GPGGHLLQTQLRGCGELHSAAASPT
GPRWTPTADTTTGLWR
GRGGHLLQTQLR
GRGGHLLQTQLRGCGELHSAAASPT
GRVDTYCR
GRVDTYCRHNYGVVESFTVQR
KCHFFNGTER
KCHFFNGTERFLDR
KCHFFNGTERVR
MERFTPAK
MERFTPAKWSTQA
RAAVDTYCR
RAAVDTYCRHNYGVVESFTVQR
RAGWTPTADTTTGLWR
RDKPSHAVEWR
RDKPSHAVEWRAR
RDKPSHSGMESTV
RECHFFNGTERFLDR
RGPGGHLLQTQLR
RGPRWTPTADTTTGLWR
RGRGGHLLQTQLR
RGRVDTYCR
RRAAVDTYCR
RRAGWTPTADTTTGLWR
RRGPGGHLLQTQLR
VDTYCRHNYGVVESFTVQR
VDTYCRHNYGVVESFTVQRR
WRGLHLPSGAPK
WRGLHLPSGAPKR
WTPTADTTTGLWR
WTPTADTTTGLWRASQCSGESNLR
YRECHFFNGTER
YRECHFFNGTERFLDR
YRECHFFNGTERVR
